doc/support/thermochem.rst

```diff
@@ -0,0 +1,4 @@
+Thermochemistry mechanism management
```

Comment:
If this file is being added, it should probably be added to a toctree as well.

Reply:
I added this under *support*, and in the toctree there already.  Is there an additional place to add it?

Reply:
Duh. Missed that. (despite it being right above this in the diff) Sorry! All good.